layouts/partials/webinar-list-item.html

```diff
@@ -1,27 +1,43 @@
-<li class="w-full md:w-1/2 lg:w-1/3 m-0 p-2">
-    <article class="rounded shadow-md bg-white border border-gray-200 mb-10 flex flex-col mx-auto" style="max-width: 320px;">
-        <div style="height: 239px;">
+<li class="w-full m-0 p-2{{ cond (eq .Params.featured true) " order-first" "" }}">
+    {{ if eq .Params.featured true }}
+        <p class="hidden text-2xl font-bold lg:block">FEATURED</p>
+    {{ end }}
+    <article class="rounded shadow-md bg-white border border-gray-200 mb-10 flex flex-col lg:flex-row mx-auto">
+        <div class="w-full {{ cond (eq .Params.featured true) "lg:w-5/12" "lg:w-4/12" }}">
             {{ if .Params.external }}
             <a href="{{ .Params.url_slug }}">
             {{ else }}
             <a class="text-2xl" href="/webinars/{{ .Params.url_slug }}/">
             {{ end }}
-                <div class="w-full h-full bg-contain bg-no-repeat bg-center bg-black" style="background-image: url('{{ .Params.preview_image }}');"></div>
+                <img class="w-full" src="{{ .Params.preview_image }}" />
             </a>
         </div>
-        <div class="h-32 p-3">
-            <h3>
+        <div class="w-full {{ cond (eq .Params.featured true) "lg:w-7/12" "lg:w-8/12" }} p-3 ml-3">
+            <div class="">
                 {{ if .Params.external }}
-                    <a class="text-xl" target="_blank" rel="noopener noreferrer" href="{{ .Params.url_slug }}">
+                    <a class="text-xl md:text-2xl font-bold" target="_blank" rel="noopener noreferrer" href="{{ .Params.url_slug }}">
                         {{ .Params.title }}
                         <i class="text-sm ml-2 fas fa-external-link-alt"></i>
                     </a>
                 {{ else }}
-                    <a class="text-xl" href="/webinars/{{ .Params.url_slug }}/">
+                    <a class="text-xl md:text-2xl font-bold" href="/webinars/{{ .Params.url_slug }}/">
                         {{ .Params.title }}
                     </a>
                 {{ end }}
-            </h3>
+                <p class="mb-5">
+                    {{ .Params.meta_desc }}
+                </p>
+                {{ if eq .Params.pre_recorded false}}
+                    <p class="webinar-datetime my-0 text-lg">
+                        <i class="fas fa-calendar mr-2"></i>
+                        <span data-datetime="{{ .Params.main.sortable_date }}">{{ .Params.main.datetime }}</span>
```

Comment:
These display as "Invalid Date" for me in Safari 13.1:

<img width="1137" alt="Screen Shot 2020-04-06 at 9 23 07 AM" src="https://user-images.githubusercontent.com/710598/78581200-6b079b00-77e8-11ea-9d26-e4f852d5f04c.png">